netci.groovy

```diff
@@ -1176,14 +1180,25 @@ combinedScenarios.each { scenario ->
                                         else if (architecture == 'x64') {
                                             if (isLongGc(scenario)) {
                                                 buildCommands += "tests\\runtest.cmd ${lowerConfiguration} ${architecture} longgctests sequential"
-                                            } else {
+                                            } 
+                                            else {
+                                                buildCommands += "tests\\runtest.cmd ${lowerConfiguration} ${architecture}"
+                                            }
+                                        }                                        
+                                        else if (architecture == 'x86ryujit') {
+                                            if (isLongGc(scenario)) {
+                                                buildCommands += "tests\\runtest.cmd ${lowerConfiguration} ${architecture} longgctests sequential"
+                                            } 
+                                            else {
```

Comment:
You also need to pass `TestEnv <full_path>\tests\x86\ryujit_x86_testenv.cmd`. Is that done here or elsewhere?